### DISCUSSION TITLE
Import already deployed helm

### DISCUSSION THREAD

**Discussion by luk-sam** [argoproj/argo-cd#8647] [Q&A] [answered]
Hi,
is there any tool or even manual way to import already deployed helm chart into application and sync?
I love argo, however without such feature it's useless in production (in our use case ofc, e.g. regarding to undeletable flux proxy)

kr,
Luk

**Comment by crenshaw-dev** [comment, marked as answer]
I'm not 100% certain, but I think if you add an Application which contains resources that are already on the cluster, Argo CD will manage those resources as if they were deployed with the Application.

However, I think there are challenges involved if you still want to use `helm` tooling for deployment alongside Argo CD. I believe that, once you start managing Helm-generated resources with Argo CD, you're meant to _only_ manage them with Argo CD.

**Reply by junoriosity** [reply]
@crenshaw-dev That sounds very interesting. Assume, for instance, that I already deployed nginx-ingress in a namespace. How would I add it then to ArgoCD?
